functions/greater-than-ten/greater-than-ten.spec.js

```diff
@@ -0,0 +1,11 @@
+const checkValue = require("./greater-than-ten");
+describe("greater-than-ten", () => {
+  it("should return true if greater 10", () => {
+    const result = checkValue(55);
+    expect(result).toBe(true);
+  });
+  it("should return true if greater 10", () => {

```

Comment:
message should be should return false if less then 10

Reply:
done